Shuttle (NEW): MOFF Atlas - Cargo/Salvage Mothership

Voltron9100 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
My Magnum Opus! The MOFF (Montegomery Olson's Freighter Fabrications) Atlas! If this gets denied I will cry and maybe pee a little, and yes it would be your fault.

## Why / Balance
I saw many people requesting a large mothership for Salvage and Cargo. This is ALSO something that I would like, so I decided to take a crack at it. It should be stated that the goal of this ship was not to minmax. I say this because I've recieved criticism on how it should be minmaxxed more, and others saying that it is TOO minmax-y. Suffice to say, I think what I have achieved, is a good balance, using the polarization of criticism as my evidence. Ultimately, the goal of the ship is very RP-Centric. I think such a vessel could create a lot of fun RP, especially considering all the gizmos I packed into it, in comparison to other ships. When I say "gizmos" I'm referring to "linked devices"

There are also some amenities that I added for the sake of visuals, and figured that such things could be easily removed, such as the booze or soda dispensers.

The zoomed out screenshots don't quite do a ship of this size justice. In game, you'll find that the flow of the layout feels natural and fitting. The living/office wing feel comforting in comparison to the industrial sections, and the industrial sections give you the perfect "I'm a cog in a corporate machine" vibe. A real Working Mans cargo bay.

Speaking of the Cargo Bay, LOOK at that puppy! There's no quite putting into words how immersive such an environment is, till you stand on it in game. On another note, such a bay alleviates "Crate-Stacking" to a large extent.

With all that said and done, I will now commence the hypnotizing subliminal messaging part of this PR to guarantee its approval.
PLEASE ACCEPT PLEASE I BEG OF YOU PLEASE IM LITERALLY CRYING RN PLEASE I LOVE YOU PLEASE

## Media
![Screenshot 2025-03-31 025057](https://github.com/user-attachments/assets/6dff1f59-941b-43c7-bf9b-046ddcff7943)
![Screenshot 2025-03-31 025129](https://github.com/user-attachments/assets/013f236c-5d7e-4ef0-b892-afbff8f31dbc)


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read [CONTRIBUTING.md](https://github.com/new-frontiers-14/frontier-station-14/blob/master/CONTRIBUTING.md) and and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I have reviewed the [Ship Submission Guidelines](https://frontierstation.wiki.gg/wiki/Ship_Submission_Guidelines) if relevant.
- [X] I confirm that AI tools were not used in generating the material in this PR.

Masterscape commented:
> If this gets denied I will cry and maybe pee a little, and yes it would be your fault.

![image](https://github.com/user-attachments/assets/27aafa1a-f417-4862-9989-164b5cb70465)
You what.

ErhardSteinhauer commented on Resources/Prototypes/_NF/Shipyard/atlas.yml:
```diff
@@ -0,0 +1,35 @@
+  parent: BaseVessel
+  name: MOFF Atlas
+  description: A massive-sized vessel specializing in long-haul salvage, mining, and cargo operations.
+  price: 100
```
A bit too low for this ship

ErhardSteinhauer commented:
![image](https://github.com/user-attachments/assets/df0b08c5-b482-4115-afea-088a42754e19)

A single line would suffice.
`- add: Added a new cargo/salvage shuttle - MOFF Atlas.`

Alkheemist commented:
![Atlas-0](https://github.com/user-attachments/assets/ec96a8a5-6d61-41e8-84d8-47a1f5297870)

have a nice maprenderer render of the ship

Voltron9100 commented:
> You mentioned that this is a mothership, does it have a mothership shipyard console? Why are there random materials mapped? Those will just get stolen + its a salvage ship so people can just salvage those easily.

It is a mothership, but I forgot to ask about adding its own console. Right now, it's just using the Cads as a placeholder. I can get this squared away with some help. Most of the crates are empty crates, and I figured it'd be nice for a larger crew to have some mats to get their ball rolling. But you're right that it's not really necessary. It's just low value starter mats. I got rid of some mats and empty/filled crates. I just think the gamble is fun. I can remove them completely though.

Voltron9100 commented:
Tweaked the air alarms at the risk of making it a little more convoluted. I generally just added a bit more where, not having specific air alarms, could impede movement to fix the issues causing the air alarms to lock everything down in the first place. Along with this, I rerouted some waste pipes to be easier to access.

Also added more shutters around the cockpit windshield, although this also required adding *another* button. Unsure if this was the move or not. Generally I just think for a ship of this size and stature, some built-in piracy protection could be warranted. After all, NT wouldn't wanna _lose_ their precious **ship!**

I mean _crew..._

Voltron9100 commented:
Committed it.

I also did a few tweaks to it here and there. Removed the unnecessary air sensors, and changed the price from 250k to 175k, after noticing that this ship is more RP driven rather than profit driven. Of course, others may feel otherwise, but I tried focusing heavily on balancing the design, without outright sacrificing it's viability, or "vibe".
